packages/tasks/src/model-libraries.ts

```diff
@@ -783,6 +783,21 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pxia,
 		filter: false,
 	},
+	"pruna-ai": {
+		prettyLabel: "Pruna AI",
+		repoName: "Pruna AI",
+		repoUrl: "https://github.com/Pruna-AI/pruna-ai",
```

Comment:
I'm getting a 404 on this link. Shouldn't it be https://github.com/PrunaAI/pruna instead? 

```suggestion
		repoUrl: "https://github.com/PrunaAI/pruna",
```